<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>951</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amazon Template</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Source template</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTo">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBrowseSource">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExtractProductInfos">
        <property name="text">
         <string>Extract product infos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>OpenAI Key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditOpenAiKey">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeViewToFill">
      <attribute name="headerDefaultSectionSize">
       <number>200</number>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeViewSources">
      <attribute name="headerDefaultSectionSize">
       <number>200</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Extra ChatGpt instructions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonViewFormatExtraInfos">
          <property name="text">
           <string>View format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClearPreviousChatGptReplies">
          <property name="text">
           <string>Clear previous ChatGpt replies</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditExtraInfos"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonGenerate">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonGenAiDesc">
          <property name="toolTip">
           <string>Generate only AI description</string>
          </property>
          <property name="text">
           <string>Generate AI Desc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRunPromptsManually">
          <property name="toolTip">
           <string>Paste prompt results manually</string>
          </property>
          <property name="text">
           <string>Run prompts manually</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonReviewAiDesc">
          <property name="toolTip">
           <string>Review AI Description and remove the failed one</string>
          </property>
          <property name="text">
           <string>Review Ai Desc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditLog"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
